src/controller/python/chip-device-ctrl.py

```diff
@@ -121,6 +121,8 @@ def ParseValueWithType(value, type):
         return value
     elif type == 'bytes':
         return ParseEncodedString(value)
+    elif type == 'bool':
+        return (value not in ["False", "false", "0"])
```

Comment:
consider using value.upper() so that things like 'FALSE' may also be detected. I would consider

```python
    return (value.upper() not in ['F', 'FALSE', '0'])
 ```